draft-ietf-quic-transport.md

```diff
@@ -678,6 +691,22 @@ packet protection in detail.  After decryption, the plaintext consists of a
 sequence of frames, as described in {{frames}}.
 
 
+## Compound Packets {#packet-compound}
+
+A sender can combine multiple QUIC packets (typically a Cryptographic Handshake
+packet and a Protected packet) into one UDP packet, thereby reducing the number
+of UDP packets required to be emitted when application data can be sent during
+the handshake.
+
+The sender MUST NOT combine QUIC packets belonging to different QUIC
+connections into a single UDP packet.
+
+Every QUIC packet that is being conveyed in a compound UDP packet is a complete
+QUIC packet.  No fields in the packet header are omitted.  The receiver of a
+compound UDP packet MUST individually process each QUIC packet and separately
+acknowledge them, as if they were received as distinct UDP packets.
```

Comment:
s/as distinct UDP packets/as the payload of different UDP datagrams/

Reply:
Done in 49d6f96.